### DISCUSSION TITLE
[ChainSync client] spend redeemers

### DISCUSSION THREAD

**Discussion by KristianBalaj** [CardanoSolutions/ogmios#306] [Q & A] [answered]
Babbage block has TXs of type `TxBabbage` where the `tx.witness.redeemers` are in the following format by this example:
```json
"redeemers": {
      "spend:0": {
        "redeemer": "2HmA",
        "executionUnits": ...
      },
      "spend:1": {
        "redeemer": "2HmA",
        "executionUnits": ...
      },
      "mint:2": {...},
      ...
    },
```

How to know which input is spent by which redeemer? Maybe the number behind the column is the index of the input in the inputs array?

**Comment by KristianBalaj** [comment, marked as answer]
Yes, it should be the index in the array of inputs.

**Reply by KtorZ** [reply]
The input index is exactly what it is indeed.
